geopyspark/tests/png_test.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 
 from geopyspark.geotrellis import Extent, ProjectedExtent, Tile
-from geopyspark.geotrellis.constants import SPATIAL, HOT
+#from geopyspark.geotrellis.constants import SPATIAL, HOT
```

Comment:
Yeah, this can be removed. However, this entire file is actually deleted in another PR #296, so I just commented it out in this one just so the tests can pass.